samples/web/content/apprtc/js/util.js

```diff
@@ -83,3 +85,97 @@ function filterTurnUrls(urls, protocol) {
     }
   }
 }
+
+// Return a random numerical string.
+function randomString(strLength) {
+  var result = [];
+  strLength = strLength || 5;
+  var charSet = '0123456789';
+  while (strLength--) {
+    result.push(charSet.charAt(Math.floor(Math.random() * charSet.length)));
+  }
+  return result.join('');
+}
+
+// Returns true if the code is running in a packaged Chrome App.
+function isChromeApp() {
+  return (typeof chrome !== 'undefined' &&
+          typeof chrome.storage !== 'undefined' &&
+          typeof chrome.storage.local !== 'undefined');
+}
+
+// Get a value from local browser storage. Calls callback with value.
+function getStorage(key, callback) {
+  if (isChromeApp()) {
+    // use chrome.storage.local
```

Comment:
Right, all comment should start with capitalized and end with '.'

On Wed, Jan 7, 2015 at 1:56 PM, Chuck Hays <notifications@github.com> wrote:

> In samples/web/content/apprtc/js/util.js
> <https://github.com/GoogleChrome/webrtc/pull/345#discussion-diff-22619522>
> :
>
> > +    result.push(charSet.charAt(Math.floor(Math.random() * charSet.length)));
> > +  }
> > +  return result.join('');
> > +}
> > +
> > +// Returns true if the code is running in a packaged Chrome App.
> > +function isChromeApp() {
> > +  return (typeof chrome !== 'undefined' &&
> > +          typeof chrome.storage !== 'undefined' &&
> > +          typeof chrome.storage.local !== 'undefined');
> > +}
> > +
> > +// Get a value from local browser storage. Calls callback with value.
> > +function getStorage(key, callback) {
> > +  if (isChromeApp()) {
> > +    // use chrome.storage.local
>
> Is this referring the the comment line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/webrtc/pull/345/files#r22619522>.
>



-- 
Jiayang Liu